### PR TITLE
Enable asset claim with Crossmint

### PR DESCRIPTION
**Pull request by gdethier**
`development.json` is now gitignored to prevent the leak of Crossmint API key. One has to create `development.json` using `development.json.sample` as a template and, optionally, set an actual API key.

logion-network/logion-internal#594

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@craco/craco": "^6.4.5",
+        "@ethersproject/transactions": "^5.7.0",
```
Comment (benoitdevos) [inline package.json:13]: Why do we need this dependency ? Is it to "force" a specific version ?

File: package.json [hunk 2]
```diff
@@ -18,6 +20,7 @@
         "crypto-browserify": "^3.12.0",
         "csv-parser": "^3.0.0",
         "detect-browser": "^5.3.0",
+        "ethers": "^5.7.1",
```
Comment (benoitdevos) [inline package.json:23]: Same question, why do we need this dependency ?

**Review by benoitdevos — APPROVED** [review]
Good job !
I have 2 questions:

File: package.json (hunk 1 above)
Comment (gdethier) [reply package.json:13]: No, I forgot to remove it after some tests. Thanks for spotting it!

File: package.json (hunk 2 above)
Comment (gdethier) [reply package.json:23]: Same as above.

**Pull request merged**